libraries/AP_Parachute/AP_Parachute.h

```diff
@@ -110,6 +121,7 @@ class AP_Parachute {
     bool        _release_initiated:1;    // true if the parachute release initiated (may still be waiting for engine to be suppressed etc.)
     bool        _release_in_progress:1;  // true if the parachute release is in progress
     bool        _released:1;             // true if the parachute has been released
```

Comment:
You should be able to remove these now. You could replace with simple functions that return bools. EG 
`_release_initiated` could be replaced by:
```
bool AP_Parachute::release_initiated() {
    switch(_release_status) {
        case NotReleased:
            return false;
        case ReleaseInitiated:
        case ReleaseInProgress:
        case Released:
            return true;

    }
}
```
I have not looked if that logic is actually correct!